From: Marcin Owsiany <marcin@owsiany.pl>
Subject: [PATCH] bugfix/hppa-build

Fix the linux on hppa build.
Remove the assumption that hp arch == hp-ux.

Signed-off-by: Marcin Owsiany <marcin@owsiany.pl>

---
 Imakefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Imakefile b/Imakefile
index 539905c..79ab857 100644
--- a/Imakefile
+++ b/Imakefile
@@ -76,8 +76,10 @@ XCOMM No space after the equals-sign
 .PRECIOUS: distrib
 
 #ifdef HPArchitecture
+#ifndef LinuxArchitecture
         CCOPTIONS = -Aa -D_HPUX_SOURCE
 #endif
+#endif
 
 #ifdef HAVE_XPM
           XPM_LIB = -lXpm
-- 
tg: (d3dafc5..) bugfix/hppa-build (depends on: upstream)
